ui/client/common/TestResultUtils.ts

```diff
@@ -91,7 +91,7 @@ class TestResultUtils {
   }
 
   private _nodeResults(results: TestResults, nodeId: NodeId): NodeResult[] {
-    return results?.nodeResults?.[nodeId] || []
+    return sortBy(results?.nodeResults?.[nodeId] || [], (x) => x.context.id)
```

Comment:
shouldn't it be sorted on BE?

Reply:
It's the same question I asked myself :D
Ok, let me change it

Reply:
Moved sorting to the backend (sort each list by context id for consistency)